tests/h/cli/commands/groups_test.py

```diff
@@ -12,13 +12,36 @@ class TestAddCommand(object):
     def test_it_creates_a_third_party_open_group(self, cli, cliconfig, group_service):
         result = cli.invoke(groups_cli.add_open_group,
                             [u'--name', 'Publisher', u'--authority', 'publisher.org',
-                             u'--creator', 'admin'],
+                             u'--creator', 'admin', u'--origin', 'publisher.org'],
                             obj=cliconfig)
 
         assert result.exit_code == 0
 
         group_service.create_open_group.assert_called_once_with(
-            name=u'Publisher', userid='acct:admin@publisher.org')
+            name=u'Publisher',
+            userid='acct:admin@publisher.org',
+            origins=('publisher.org',),
+        )
+
+    def test_it_accepts_multiple_origin_options_on_the_command_line(self,
+                                                                      cli,
+                                                                      cliconfig,
+                                                                      group_service):
+        cli_options = [
+            u'--name', 'Publisher',
+            u'--authority', 'publisher.org',
+            u'--creator', 'admin',
+            u'--origin', 'hostname1.org',
```

Comment:
Ah, these `origin`s lack scheme or ports—makes me think back to my question about whether an example should be provided for CLI options...

Reply:
Oops, no I change my mind that the origins _should_ have schemes. Fixed